dev/otel-agent-config.yaml

```diff
@@ -0,0 +1,38 @@
+receivers:
+  jaeger:
+    protocols:
+      thrift_compact: # on port 6831
+      thrift_binary: # on port 6832
+
+processors:
+  batch:
+  attributes:
+    actions:
+    - key: apiKey
```

Comment:
If these need redacting you need to PR against https://github.com/GaloyMoney/galoy-infra/blob/main/modules/services/opentelemetry-values.yml as well